src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -43,7 +44,8 @@ static RecordableCommand()
                 jsonSettings = new JsonSerializerSettings()
                 {
                     TypeNameHandling = TypeNameHandling.Objects,
-                    ContractResolver = new CamelCasePropertyNamesContractResolver()
+                    ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                    Culture = CultureInfo.InvariantCulture
```

Comment:
hmmm!
I'm not sure about this change. I'm not sure this has been touched in years?